SU2_CFD/src/variables/CHeatFVMVariable.cpp

```diff
@@ -39,7 +39,7 @@
 
 
 CHeatFVMVariable::CHeatFVMVariable(su2double heat, unsigned long npoint, unsigned long ndim, unsigned long nvar, CConfig *config)
-  : CVariable(npoint, ndim, nvar, config) {
+  : CVariable(npoint, ndim, nvar, config), Gradient_Reconstruction{config->GetReconstructionGradientRequired() ? Gradient_Aux : Gradient} {
```

Comment:
You did not post the complete error message... But I think I got it: https://gcc.godbolt.org/z/pUhCEu
I think it is a compiler bug, it is recognizing the {...} as an initializer list (which it is not) can you try changing it to (...) instead and see if that compiles?
Also... maybe update your compiler xD

Reply:
Yep, normal braces are doing the trick! Thanks a lot (and sorry for beeing so lazy. That Website directly went to my bookmarks). Does anything speak against changing to normal brace? I can also do that and push... also for the cosmetic stuff @economon 

> Also... maybe update your compiler xD

Yeah. I think that every other day, but installing that on our cluster is not that straight forward for people with limited skillz :(

Reply:
Initially I tried normal braces and had some issue which I could not replicate with that example. I'll have a closer look. (I figured it would be a cluster thing)

Reply:
Any news on this issue? I think it is good that you are testing on older compilers, since unfortunately, many folks will be constrained by older systems, especially clusters. Portability and easy install are critically important for us.

Reply:
sorted

Reply:
> sorted? 

Is it ok then to change to normal braces? I tried multiple versions of gcc & clang with your provided link and couldn't and didn't get any errors with normal braces. Or do have issues with normal braces